components/automate-ui/src/app/entities/environments/environment.model.ts

```diff
@@ -1,3 +1,38 @@
+import { merge } from 'lodash';
```

Comment:
Try to use only `lodash/fp` when bringing in new functionality. (Consider `lodash` deprecated by convention.)

Reply:
I have added this :)